src/v2/Apps/Artist/routes.tsx

```diff
@@ -93,6 +94,17 @@ export const routes: RouteConfig[] = [
         }
       }
     `,
+    getCacheConfig: ({ context }) => {
+      if (isServer) {
+        // If there's a user, request new data; otherwise check to see if there
+        // is a cache available in the relay network layer
+        const fetchData = !!context.user ? true : false
+
+        return {
+          force: fetchData,
+        }
+      }
+    },
```

Comment:
Here's an example of how we would leverage our route configs to achieve fine-grained cache control on the server, which follows Relay's API. 

If we wanted to achieve more sophisticated control we could perform further operations during the server-render pass and pass the [result into our context](https://github.com/artsy/force/blob/master/src/v2/Artsy/Router/buildServerApp.tsx#L81). (This can also happen [on the client](https://github.com/artsy/force/blob/master/src/v2/Artsy/Router/buildClientApp.tsx#L87).)

What immediately comes to mind is improving the logged-in performance for all non-auction / BNMO artworks as described here https://github.com/artsy/reaction/pull/3164/)

Reply:
Yea, it seems like doing it this way is a good start and we can iterate with the finer grained controls.

Reply:
This has been updated so we always perform a fetch on the server if there's a user, versus having to define the rule on each route. We can fine tune this later.